views/administration/users_edit.hbs

```diff
@@ -149,7 +149,7 @@
                         {{/ifCond}}
                         {{#if canDeleteUser}}
                         <button style="margin: 0 10px 20px 0"
-                            class="btn btn-secondary btn-delete">{{$t "administration.users_edit.headline.deleteUser" }}</button>
+                            class="btn btn-secondary btn-delete" data-testid="button_delete-user_submit">{{$t "administration.users_edit.headline.deleteUser" }}</button>
```

Comment:
dont know if mixing "-" and "_" in one id is a good idea, that could lead to typos

Reply:
I agree. Did this because of data-testids like: input_create-user_firstname and working from the tests. Will change my new ids to all use "_".